build/constants.js

```diff
@@ -14,6 +14,7 @@ var DAO_ENTITIES = exports.DAO_ENTITIES = {
   KEGS: 'kegs',
   LOCATIONS: 'locations',
   PERMISSIONS: 'permissions',
+  POUR_CHART: 'chart',
```

Comment:
not sure about the names of all of this... Here `chart` is an actual endpoint, and it doesn't match with the key, because its is not very descriptive.I hope we will have more charts and we change the endpoint then.